corehq/apps/locations/tests/util.py

```diff
@@ -62,27 +62,31 @@ def create_locations(locations, types, parent):
     return locations_dict
 
 
-def setup_locations_and_types(domain, location_types, locations):
+def setup_locations_and_types(domain, location_types, stock_tracking_types, locations):
```

Comment:
nit: i think it'd be easier to reason with this function if location_types were specs that also included whether or not they were stock tracking. seems a little odd to have the separate params. also I think this will crash if you pass something into `stock_tracking_types` that isn't in `location_types`. 

it's just test code so seems fine to leave unless you feel like changing